components/ProgressView/examples/ProgressViewExample.m

```diff
@@ -170,13 +185,14 @@ - (void)setupConstraints {
     @"backwardAnimateLabel" : _backwardProgressAnimateLabel,
   };
   NSDictionary *metrics = @{
-    @"p" : @20,
+    @"t" : @20,
```

Comment:
Does this work correctly with the in-call status bar? Is there a way we could work with the `safeAreaInsets` or `safeAreaLayoutGuide` instead?

Reply:
It does work with the status bar. I looked into safeAreaInsets but ended up using `layoutMargins` in ecb2c39. Does that work for you? I tested it again with iPhone 8 in iOS 11 and iPhone X in iOS 11 and 12.